site/src/routes/apps/index.json.js

```diff
@@ -0,0 +1,20 @@
+import send from '@polka/send';
+import { query } from '../../utils/db';
+
+export async function get(req, res) {
+	if (req.user) {
+		const offset = req.params.offset || 0;
+		const rows = await query(`
```

Comment:
Everywhere else you're ensuring `uid` is not in the hyphenated format – that's lacking here.

Refresher:

```js
uid: row.uid.replace(/-/g, ''),
```

Reply:
good catch, thanks 👍